src/python/ensembl/io/genomio/gff3/simplifier.py

```diff
@@ -116,7 +116,19 @@ def __init__(
 
         # Init the actual data we will store
         self.records = Records()
-        self.annotations = FunctionalAnnotations()
+        self.annotations = FunctionalAnnotations(self.get_provider_name())
+
+    def get_provider_name(self) -> str:
```

Comment:
```suggestion
    def get_provider_name(self) -> str:
    """Returns the provider name for this genome.

    If this information is not available, will try to infer it from the assembly accession. Will
    return "GenBank" otherwise.
    """
```